hack/deploy-logging.sh

```diff
@@ -218,23 +220,19 @@ construct_image_name() {
 update_images_in_clo_yaml() {
     local yamlfile=$1
     local clo_img=${2:-} #unused.  leaving for compatibility
-    local version=${3:-latest}
+    local version=${3:-latest} #unused. leaving for compatibility
```

Comment:
Why leaving for compatibility? You're ignoring the argument, why create an unused local variable?